docs/source/publish_and_share_kedro_viz_on_azure.md

```diff
@@ -0,0 +1,121 @@
+
+Sign in to the [AzurePortal](https://portal.azure.com/#home) and create an App registration.
+For more information, see the official Azure documentation about [App Registration](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app).
+
+![](./images/azure_app_registration.png)
+
+When registration finishes, the app registration's Overview pane is displayed. You see the Application (Client) ID and Directory (Tenant) ID. We will use these values to set our environment variables.
+
+![](./images/azure_app_secrets.png)
+
+Add a Client Secret for the app registration. For more information, see [Add a client secret](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-a-client-secret)
+
+![](./images/azure_client_secret.png)
+
+Once the Client Secret is created, the Client secrets section is displayed and you can find the Client Secret Value as shown below
+
+![](./images/azure_client_secret_value.png)
+
+Assign Access Control (IAM) role to the storage account. For more information, see [Assign Azure roles](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=delegate-condition)
+
+Go to the storage account that is created and click on Access control (IAM) as shown below
+
+![](./images/azure_iam_tab.png)
+
+Add role assignment and select the role `Storage Blob Data Contributor` as shown below
+
+![](./images/azure_add_role_assign.png)
+
+![](./images/azure_storage_role.png)
+
+On the Members tab, select User, group, or service principal to assign the selected role to the app registration. Click on Select members, and find your app registration name by typing in the Select box as shown below 
+
+![](./images/azure_member_assign.png)
+
+## Step 3: Set credentials
+
+Once that's completed, you'll need to set your Azure credentials as environment variables in your terminal window, as shown below:
+
+```bash
+export AZURE_STORAGE_TENANT_ID="your-app-tenant-id"
+export AZURE_STORAGE_CLIENT_ID="your-app-client-id"
+export AZURE_STORAGE_CLIENT_SECRET="your-app-client-secret-value"
+```
+
+For more information, see the official Azure documentation about [how to work with environmental credentials](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.environmentcredential?view=azure-dotnet).
+
+```{note}
+Kedro-Viz uses Gen2 filesystem protocol `abfs` to write files on AzureBlobStorage.
+```
+
+```{important}
+Having a `$web` container in your AzureBlobStorage is mandatory to use Kedro-Viz publish and share feature on Azure. For more information, see the official Azure documentation about 
+[Setting up a static website](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website#setting-up-a-static-website).
+```
```

Comment:
Can we move the above `{note}` and `{important}` sections before `## Step 3: Set credentials` ?

Reply:
Resolved.